### ISSUE TITLE
Our 'for staff' contributors guide doesn't mention the 'github for newcomers' guide

### ISSUE THREAD

**Issue by bvhme** [publiccodenet/about#704]
It would make sense if [`for-staff`](https://github.com/publiccodenet/about/blob/develop/contributor-guides/for-staff.md) included a link to [`github-for-newcomers`](https://github.com/publiccodenet/about/blob/develop/activities/trainings/github-for-newcomers.md)

For instance:

```markdown
* Github - push a branch, make a pull request (maybe to improve this page!). If you are new to this, [take some basic training](../activities/trainings/github-for-newcomers.md). Then start with a '[good first issue](https://github.com/publiccodenet/about/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)'. You can request an invitation to have your Github account added to the [Foundation for Public Code](https://github.com/publiccodenet/) organizational account.
```

**Comment by himanshu007-creator** [comment]
Please assign me this issue🙂

**Comment by ElenaFdR** [comment]
Solved with #904. Thanks again, @himanshu007-creator!
